packages/framework/client-logger/fluid-telemetry/README.md

```diff
@@ -62,17 +73,25 @@ startTelemetry(telemetryConfig);
 // Done! Your container telemetry is now being created and sent to your Telemetry Consumer which will forward it to Azure App Insights.
 ```
 
-Congrats, that's it for now! If you've decided to use Azure App Insights, we have designed useful prebuilt queries for you that utilize the generated telemetry
+That's it for now! If you've decided to use Azure App Insights, we have designed useful prebuilt queries that utilize the generated telemetry. You can try out these queries after running your application with above code for a little duration.
 
-## Example 2: Extending ITelemetryConsumer to send Fluid telemetry to a custom destination
+You can find these queries [in the "Interpreting Telemetry Data" section below.](#telemetry_visualization).
 
-In this example, you'll walk through the basic setup process to start getting container telemetry to be produced and logging it to the console.
+## Use Case 2: Setup a custom telemetry consumer
 
-### Step 1: Install the @fluidframework/fluid-telemetry package dependency
+To use a custom telemetry consumer and integrate with a cloud provider or custom dashboard solution, you need to implement the `ITelemetryConsumer` interface.
+
+In this example, you'll walk through the basic setup process to start getting container telemetry to be produced and log it to the console.
+
+### Step 1: Install new package dependencies
+
+Install the Fluid Framework telemetry package dependency
 
 -   Using NPM: `npm install @fluidframework/fluid-telemetry`
 
-### Step 2: First, we'll have to create our own telemetry consumer which extends the ITelemetryConsumer interface. Let's look at an example that will simply console.log the telemetry.
+### Step 2: Define a custom telemetry consumer
+
+Here we'll create our own telemetry consumer which extends the `ITelemetryConsumer` interface. Let's look at an example that will simply log the telemetry to console.
```

Comment:
This was not part of your changes but on line 113 the comment should be changed to `// 1: import our implementation of MySimpleTelemetryConsumer from step 2` (right now it says step 1)